compiler/parser/parser.peg

```diff
@@ -478,6 +479,23 @@ SampleExpr
   = _ lval:Lval { RETURN(lval)}
   / "" { RETURN(MAP("kind":"ID", "name":"this")) }
 
+LoadOp
+    = "load" _ pool:PoolNameString author:AuthorArg? message:MessageArg? meta:MetaArg? branch:BranchSpecification? {
```

Comment:
Put branch immediately after pool so the syntax is "pool@branch", which matches what we do with PoolCommit.
```suggestion
    = "load" _ pool:PoolNameString branch:BranchSpecification? author:AuthorArg? message:MessageArg? meta:MetaArg?  {
```